package.json

```diff
@@ -39,7 +39,9 @@
     "docs": "rm -rf docs && ./node_modules/.bin/jsdoc -r -c ./jsdoc_conf.json -t ./node_modules/ink-docstrap/template -d docs/ -R README.md",
     "docs:deploy": "./node_modules/.bin/gh-pages -d docs",
     "gh-pages-travis": "gh-pages-travis",
-    "commit": "./node_modules/.bin/git-cz",
+    "cz": "./node_modules/.bin/git-cz",
+    "precommit": "npm run validate",
```

Comment:
__IF__ we should lint and test, while using git, it should be done on pre-push.

Right now, if I write a bad message, it first lints and tests and only afterwards, it tells me that I didn't write the message correctly.